src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1454,6 +1482,7 @@ setup_TxQ(Config const& config)
     TxQ::Setup setup;
     auto const& section = config.section("transaction_queue");
     set(setup.ledgersInQueue, "ledgers_in_queue", section);
+    set(setup.queueSizeMin, "minimum_queue_size", section);
```

Comment:
Should this be documented in `rippled-example.cfg`

Reply:
Probably, yeah.